tools/stats/scribe.py

```diff
@@ -0,0 +1,33 @@
+import os
+import json
+
+import requests  # type: ignore[import]
+import boto3  # type: ignore[import]
+
+
+def send_to_scribe(logs: str) -> str:
+    access_token = os.environ.get("SCRIBE_GRAPHQL_ACCESS_TOKEN", "")
+    if access_token == "":
+        return _send_to_scribe_via_boto3(logs)
+    return _send_to_scribe_via_http(access_token, logs)
+
+
+def _send_to_scribe_via_boto3(logs: str) -> str:
+    print("Scribe access token not provided, sending report via boto3...")
+    data = {"logs": logs},
+    client = boto3.client("lambda")
+    res = client.invoke(FunctionName='gh-ci-scribe-proxy', Payload=json.dumps(data).encode())
+    payload = str(res['Payload'].read().decode())
+    if res['FunctionError']:
+        raise Exception(payload)
```

Comment:
i've not read boto3 client doc but is this payload the same as the json dump data? if so this might be a huge error msg yes?

Reply:
yes, this is the same message as the `via_http` route that does `print(r.text)`, which is what we are currently doing

Reply:
yeah I always found this weird but let's address this separately.